components/common/Layout/Layout.tsx

```diff
@@ -31,22 +31,31 @@ const dynamicProps = {
 
 const SignUpView = dynamic(
   () => import('@components/auth/SignUpView'),
-  dynamicProps
+  {
+    ...dynamicProps
```

Comment:
For the cases that we don't need to extend the object can you leave it like before? Or does it have any problem with it?

Reply:
`dynamic()` needs an object literal, the way it was before was causing the issue